lib/src/ast/sass/argument.dart

```diff
@@ -30,6 +31,9 @@ class Argument implements SassNode {
   String get originalName =>
       defaultValue == null ? span.text : declarationName(span);
 
+  FileSpan get nameSpan =>
+      defaultValue == null ? span : span.subspan(0, name.length + 1);
```

Comment:
It's possible (albeit not very likely) that `name` has escape sequences, which would make this computation invalid. I'd probably write an `initialIdentifier()` helper in `SpanExtensions` that just walks forward through identifier characters or backslash-escapes.